log.ml

```diff
@@ -143,7 +143,7 @@ let sexp_tactic_log taclog =
     Fake_log -> simple "Fake_log"
   | Label_tac_log (st, th) -> Snode [Sleaf "Label_tac_log"; Sleaf st; sexp_thm th]
   | Accept_tac_log th -> thm "Accept_tac_log" th
-  | Conv_tac_log c -> Snode [Sleaf "Conv_tac_log"; sexp_conv c]
+  | Conv_tac_log (s, c) -> Snode [Sleaf "Conv_tac_log"; Sleaf s; sexp_conv c]
```

Comment:
This will break any S-expr parser that we write, since the leaf strings aren't escaped.  Let's leave it out for now.